app/api/chat.py

```
@@ -38,4 +38,6 @@ async def chat_endpoint(
         return ChatMessageResponse(reply=reply, thread_id=thread_id)
     except Exception as e:
         logger.exception(f"Unhandled exception in chat endpoint for thread_id {thread_id}: {e}")
-        raise HTTPException(status_code=500, detail="Internal server error") 
\ No newline at end of file
+        raise HTTPException(status_code=500, detail="Internal server error") 
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)